code/datums/components/tackle.dm

```diff
@@ -476,6 +478,11 @@
 			if(messes.len >= MAX_TABLE_MESSES)
 				break
 
+	// Kepori have mastered the art of the table-tackle and will instead fly over gracefully (most of the time)
+	if(iskepori(owner))
+		// However if there is too much junk on the table, it can't be helped
+		if(rand(1, MAX_TABLE_MESSES) > messes.len)
+			return
```

Comment:
if possible i'd like this to be moved to something like `datum/species/proc/tackle_act()`

Reply:
Wasn't sure if tackling warranted dedicated procs on the species datum but that's certainly something I can do.